### ISSUE TITLE
Build fails when source is under a directory with spaces in the name

### ISSUE THREAD

**Issue by blowdart** [aspnet/Mvc#2082]
Repo steps

1) Create a directory "Bad Example"
2) Clone MVC into the MVC directory under "Bad Example"
3) Run build
4) Watch it barf when validating node signatures because directories aren't quoted when passed to signtool.
```
D:\Bad Dir\mvc [dev +1 ~0 -0 !]> .\build
Installing 'KoreBuild 0.2.1-beta4-10103'.
Successfully installed 'KoreBuild 0.2.1-beta4-10103'.
Installing 'Sake 0.2'.
Successfully installed 'Sake 0.2'.
Determining latest version
kre-clr-win-x86.1.0.0-beta4-11138 already installed.
Adding C:\Users\bdorrans\.k\runtimes\kre-clr-win-x86.1.0.0-beta4-11138\bin to process PATH
Adding C:\Users\bdorrans\.k\runtimes\kre-clr-win-x86.1.0.0-beta4-11138\bin to user PATH
Updating alias 'default' to 'kre-clr-win-x86.1.0.0-beta4-11138'
kre-coreclr-win-x86.1.0.0-beta4-11138 already installed.
Adding C:\Users\bdorrans\.k\runtimes\kre-coreclr-win-x86.1.0.0-beta4-11138\bin to process PATH
Adding C:\Users\bdorrans\.k\runtimes\kre-clr-win-x86.1.0.0-beta4-11138\bin to process PATH
info: Target default
info: Target pre-clean
info: Target target-dir-clean
info: Target build-clean
info: Target clean
info: Target post-clean
info: Target validate
info: Target restore-npm-modules
info: Testing for command: node --version
'node' is not recognized as an internal or external command,
operable program or batch file.
warn:    command not found (non-0 exit code)
info: Installing nodejs locally
info: Downloading http://nodejs.org/dist/v0.10.28/node.exe to D:\Bad Dir\mvc\bin\nodejs\node.exe
info: Downloading http://nodejs.org/dist/npm/npm-1.4.9.zip to D:\Bad Dir\mvc\bin\nodejs\npm-1.4.9.zip
info: Unzipping npm to D:\Bad Dir\mvc\bin\nodejs
info: Verifying Authenticode signature of D:\Bad Dir\mvc\bin\nodejs\node.exe
SignTool Error: File not found: D:\Bad
SignTool Error: File not found: Dir\mvc\bin\nodejs\node.exe
warn: The signature verification for D:\Bad Dir\mvc\bin\nodejs\node.exe failed:

Number of files successfully Verified: 0
Number of warnings: 0
Number of errors: 2

verbose: Stack trace:
   at Viewb61b022fbfc04dc394e8d79a0f6fac85.<RenderViewLevel0>b__4f() in c:\Users\bdorrans\AppData\Local\Temp\cedcbae078f
b40bc83ba913e6cbaeefd-1.cs:line 1707
   at Sake.Engine.Builder.BuilderBase`1.CallTarget(String name)
   at Sake.Engine.Builder.BuilderBase`1.StartingTarget(String name)
   at Viewb61b022fbfc04dc394e8d79a0f6fac85.<RenderViewLevel0>b__51() in c:\Users\bdorrans\AppData\Local\Temp\cedcbae078f
b40bc83ba913e6cbaeefd-1.cs:line 3012
   at Sake.Engine.Builder.BuilderBase`1.CallTarget(String name)
   at Sake.Engine.Builder.BuilderBase`1.StartingTarget(String name)
   at Viewb61b022fbfc04dc394e8d79a0f6fac85.<RenderViewLevel0>b__42() in c:\Users\bdorrans\AppData\Local\Temp\cedcbae078f
b40bc83ba913e6cbaeefd-1.cs:line 1260
   at Sake.Engine.Builder.BuilderBase`1.CallTarget(String name)
   at Sake.Engine.Builder.BuilderBase`1.StartingTarget(String name)
   at Viewb61b022fbfc04dc394e8d79a0f6fac85.<RenderViewLevel0>b__43() in c:\Users\bdorrans\AppData\Local\Temp\cedcbae078f
b40bc83ba913e6cbaeefd-1.cs:line 1277
   at Sake.Engine.Builder.BuilderBase`1.CallTarget(String name)
   at Sake.Engine.Builder.BuilderBase`1.StartingTarget(String name)
   at Viewb61b022fbfc04dc394e8d79a0f6fac85.<RenderViewLevel0>b__44() in c:\Users\bdorrans\AppData\Local\Temp\cedcbae078f
b40bc83ba913e6cbaeefd-1.cs:line 1294
   at Sake.Engine.Builder.BuilderBase`1.CallTarget(String name)
   at Sake.Engine.Builder.BuilderBase`1.StartingTarget(String name)
   at Viewb61b022fbfc04dc394e8d79a0f6fac85.<RenderViewLevel0>b__45() in c:\Users\bdorrans\AppData\Local\Temp\cedcbae078f
b40bc83ba913e6cbaeefd-1.cs:line 1311
   at Sake.Engine.Builder.BuilderBase`1.CallTarget(String name)
   at Viewb61b022fbfc04dc394e8d79a0f6fac85.<RenderViewLevel0>b__3d() in c:\Users\bdorrans\AppData\Local\Temp\cedcbae078f
b40bc83ba913e6cbaeefd-1.cs:line 1167
   at Sake.Engine.Builder.BuilderBase`1.CallTarget(String name)
   at Sake.Engine.SakeEngine.Execute(Options options)
   at Sake.Engine.SakeEngine.Execute(String[] args)
   at Sake.Program.Main(String[] args)
D:\Bad Dir\mvc [dev +1 ~0 -0 !]>
```

**Comment by yishaigalatzer** [comment]
@blowdart seems to be a sake thing, not an MVC thing. Or am I wrong?

**Comment by blowdart** [comment]
I honestly don't know - building MVC is the first one that failed for me, hence you getting it. HttpAbstrations, Security, Identity all build just fine from the same directory structure.

**Comment by dougbu** [comment]
> seems to be a sake thing, not an MVC thing

neither a MVC nor Sake thing.  fix will be in the `*.shade` files from the aspnet/Universe repo.  tracking here because it shows up in our builds.

**Comment by dougbu** [comment]
fixed in commit aspnet/Universe@90294aa
